src/main/resources/resources/liquidbounce/lang/ru_ru.json

```
@@ -632,5 +632,6 @@
   "liquidbounce.scripts.debug.inspect.url": "Нажмите, чтобы скопировать URL отладчика и вставьте его в браузер (к сожалению, работают только браузеры на основе Chromium).",
   "liquidbounce.scripts.debug.support": "Скрипт %s настроен на запуск с поддержкой отладчика на: ",
   "liquidbounce.tooltip.clickToCopy": "Кликни для копирования",
-  "liquidbounce.unsupportedDeepLearning": "❌ Движок нейросети не инициализирован. Твоя система, возможно, не поддерживается."
+  "liquidbounce.unsupportedDeepLearning": "❌ Движок нейросети не инициализирован. Твоя система, возможно, не поддерживается.",
+  "liquidbounce.module.goldSrcMovement.description": "Движение Half‑Life 1 (GoldSrc): трение/ускорение HL1, классический A/D бхоп, лестницы и вода как в HL."
```

```suggestion
  "liquidbounce.module.goldSrcMovement.description": "Движение как в Half‑Life 1 (GoldSrc): трение/ускорение HL1, классический A/D бхоп, лестницы и вода как в HL."
```

_Passing russian-speaking here._